### DISCUSSION TITLE
About discord-rpc

### DISCUSSION THREAD

**Discussion by Lastorder-DC** [OpenRCT2/OpenRCT2#20156] [Q&A]
1. discord-rpc is deprecated [about 3 years ago](https://github.com/discord/discord-rpc#deprecation-notice), and replaced with [GameSDK](https://discord.com/developers/docs/game-sdk/sdk-starter-guide).
2. But OpenRCT2 still uses it with custom(outdated) [rapidjson fork](https://github.com/janisozaur/rapidjson). - Although wiki says they never bother to fix issue, but [linked issue](https://github.com/Tencent/rapidjson/issues/1205) is fixed 5 years ago.
3. Cloning `discord-rpc` to project root no longer works (at lease on Ubuntu) - https://github.com/OpenRCT2/OpenRCT2/issues/17448

I think we should replace, or remove discord-rpc.

**Comment by IntelOrca** [comment]
I believe part of the problem was that the new discord game SDK was not GPL compatible. Relevant issue: https://github.com/86Box/86Box/issues/1606

**Reply by Lastorder-DC** [reply]
Is it possible to recreate one using plugin API? If so, discord-rpc can be removed like twitch one.
